server/migrations/20230523163856-remove-OtherUnexpectedBehaviorText.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+const populateData = require('../services/PopulatedData/populateData');
+const { updateTestPlanRun } = require('../models/services/TestPlanRunService');
+const {
+    updateTestPlanReport
+} = require('../models/services/TestPlanReportService');
+const conflictsResolver = require('../resolvers/TestPlanReport/conflictsResolver');
+const BrowserLoader = require('../models/loaders/BrowserLoader');
+const AtLoader = require('../models/loaders/AtLoader');
+
+module.exports = {
+    up: queryInterface => {
+        return queryInterface.sequelize.transaction(async transaction => {
+            const testPlanRunQuery = await queryInterface.sequelize.query(
+                `SELECT id, "testResults" FROM "TestPlanRun"`,
+                {
+                    transaction
+                }
+            );
+
+            const testPlanReportQuery = await queryInterface.sequelize.query(
+                `SELECT id, status FROM "TestPlanReport"`,
+                {
+                    transaction
+                }
+            );
+
+            const atLoader = AtLoader();
+            const browserLoader = BrowserLoader();
+            const testPlanRunData = testPlanRunQuery[0];
+            const testPlanReportsData = testPlanReportQuery[0];
+            if (!testPlanRunData) {
+                // eslint-disable-next-line no-console
+                console.info('The test Results are empty');
+                return;
+            }
+
+            for (let i = 0; i < testPlanRunData.length; i += 1) {
+                const testPlanRun = testPlanRunData[i];
+                const testPlanRunId = testPlanRun.id;
+                let needsUpdate = false;
+
+                let updateParams = {
+                    testResults: testPlanRun.testResults
+                };
+
+                // eslint-disable-next-line no-console
+                console.info(
+                    `=== Id for TestPlanRun results:${testPlanRunId} ===`
+                );
```

Comment:
This log may be unnecessary as it's just logging all the known TestPlanRun ids found in the database. I've suggested an alternative below.